src/Makefile

```diff
@@ -88,7 +88,8 @@ goto-cc.dir: languages goto-programs.dir linking.dir
 
 memory-analyzer.dir: util.dir goto-programs.dir ansi-c.dir
 
-symtab2gb.dir: util.dir goto-programs.dir json-symtab-language.dir
+symtab2gb.dir: util.dir goto-programs.dir langapi.dir linking.dir \
+               json.dir json-symtab-language.dir
```

Comment:
This seems a bit sketchy. `json-symtab-language.dir` should already depend on `json.dir` right? And `langapi.dir` and `linking.dir` should already be included in `goto-programs.dir`?

Reply:
No, there isn't really that kind of dependency: it is perfectly safe to build, e.g., `json-symtab-language`, just the linking will fail. And that's really the only dependencies that are expressed here.